content/english/java-tictactoe/answer-key.md

```diff
@@ -230,7 +230,62 @@ public class TicTacToe_Nuevo {
 
 # activity-5: AI Computer Move
 ```java
-// TODO
+        public static int getComputerMoveAI(String[] curBoard){
```

Comment:
It would be nice to have comments throughout the code for others to understand/debug/explain to the kids. Up to you if you want to do it as a separate PR

Reply:
added!